llvm/include/llvm/Analysis/TargetTransformInfoImpl.h

```diff
@@ -74,6 +74,11 @@ class TargetTransformInfoImplBase {
   unsigned getInliningCostBenefitAnalysisProfitableMultiplier() const {
     return 8;
   }
+  int getInliningLastCallToStaticBonus() const {
+    // This is the value of InlineConstants::LastCallToStaticBonus before this
+    // function was introduced.
+    return 15000;
```

Comment:
Can you just use the constant enum instead of hardcoding it with comment?

Reply:
I removed the constant in this PR.